packages/pigeon/pigeons/message.dart

```diff
@@ -28,6 +28,7 @@ abstract class NestedApi {
 
 void setupPigeon(PigeonOptions options) {
```

Comment:
If these are examples, should the word example appear in the path somewhere?

Reply:
Those are actually used for tests in `run_tests.sh`.  In a later PR i've added an examples directory.  It will be a good idea to move these or something like them to examples.

Reply:
doing it down the diffbase chain is fine